packages/react/src/components/form/form-username.test.tsx

```diff
@@ -0,0 +1,153 @@
+import { PersonHandle } from "@slashid/slashid";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { vi, describe, afterEach } from "vitest";
+import { Form } from ".";
+import { TEXT } from "../text/constants";
+import { STORAGE_LAST_HANDLE_KEY } from "../../hooks/use-last-handle";
+import { createTestUser, inputUsername, MockSlashID } from "../test-utils";
+
+import { TestSlashIDProvider } from "../../context/test-providers";
+import { ConfigurationProvider } from "../../context/config-context";
+import { FactorConfiguration } from "../../domain/types";
+
+describe("#Form - username handle type", () => {
+  test("username tab should not render unless explicitly configured using allowedHandleTypes", () => {
+    const logInMock = vi.fn(async () => createTestUser());
+    const factors: FactorConfiguration[] = [
+      { method: "email_link" },
+      { method: "password" },
+    ];
+
+    render(
+      <TestSlashIDProvider sdkState="ready" logIn={logInMock}>
+        <ConfigurationProvider factors={factors}>
+          <Form />
+        </ConfigurationProvider>
+      </TestSlashIDProvider>
+    );
+
+    expect(
+      screen.queryByTestId("sid-tabs-trigger-username")
+    ).not.toBeInTheDocument();
+  });
+
+  test("username tab should not render unless explicitly configured using allowedHandleTypes", () => {
```

Comment:
"username tab should render when explicitly configured using allowedHandleTypes"

Reply:
thanks, fixed!